docs/references/config/config.toml.md

```diff
@@ -0,0 +1,2055 @@
+by the node while inbound connections are initiated by a remote party.
+
+Nodes on non-routable networks have to set their gateway to port-forward the P2P port for inbound connections to reach
+the node. Inbound connections can be accepted as long as the node has an address accessible from the Internet (using NAT or other methods).
+Refer to the [p2p.external_address](#p2pexternal_address) configuration for details.
+
+### p2p.max_num_outbound_peers
+
+Maximum number of outbound peers,
+that is, peers to which the node dials and establishes connections.
+
+```toml
+max_num_outbound_peers = 10
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+The [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) and
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers) values
+work together to define how many P2P connections the node will
+maintain at maximum capacity.
+
+The `p2p.max_num_outbound_peers` configuration should be seen as the target
+number of outbound connections that a node is expected to establish.
+While the maximum configured number of outbound connections is not reached,
+the node will attempt to establish connections to potential peers.
+
+This configuration only has effect if the [PEX reactor](#p2ppex) is enabled.
+Nodes configured as [persistent peers](#p2ppersistent_peers) do not count towards the
+configured `p2p.max_num_outbound_peers` limit
+(refer to [issue 1304](https://github.com/cometbft/cometbft/issues/1304) for more details).
+
+The connections are bidirectional, so any connection can send or receive messages, blocks, and other data. The separation into
+inbound and outbound setting only distinguishes the initial setup of the connection: outbound connections are initiated
+by the node while inbound connections are initiated by a remote party.
+
+Nodes on non-routable networks have to set their gateway to port-forward the P2P port for inbound connections to reach
+the node. Outbound connections can only be initiated to peers that have addresses accessible from the Internet (using NAT or other methods).
+Refer to the [p2p.external_address](#p2pexternal_address) configuration for details.
+
+### p2p.unconditional_peer_ids
+
+List of node IDs that are allowed to connect to the node even when connection limits are exceeded.
+
+```toml
+unconditional_peer_ids = ""
+```
+
+| Value type          | string (comma-separated)         |
+|:--------------------|:---------------------------------|
+| **Possible values** | comma-separated list of node IDs |
+|                     | `""`                             |
+
+If a peer listed in this property establishes a connection to the node, it will be accepted even if the
+configured [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) limit was reached.
+Peers on this list also do not count towards the
+configured [`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers) limit.
```

Comment:
Now I'm confused: unconditional peers are not mentioned in the `p2p.max_num_outbound_peers` section (only persistent peers are mentioned there)

Reply:
Can be dealt with later. Resolving